packages/Python/lldbsuite/test/lang/swift/implementation_only_imports/ReadMe.md

```diff
@@ -0,0 +1,11 @@
+These tests concern the experimental Swift feature `@_implementationOnly import`.\* Unlike a normal import, an `@_implementationOnly` import is guaranteed not to be part of a library's API or ABI, which means that client apps don't have to know about the import at all to use the library. However, the import *can* be used in the *implementation* of the library, so it's important for LLDB to handle the cases where the `@_implementationOnly` import is and isn't available when debugging.
```

Comment:
An even better location for this text is as a docstring in the test function (or class) in the .py file. IIRC, the test driver knows about them, too.

Reply:
This is one level up in a folder of four separate tests. I don't want to duplicate it in several places, but I mentioned it in the docstrings for the tests.